crd-convert/Makefile

```diff
@@ -0,0 +1,17 @@
+PROJECT_NAME=crd-convert
```

Comment:
We don't need this unless/until we're implementing a conversion webhook.

Reply:
Deleting whole make file?

Reply:
Did I comment this on the makefile? IDK why/who I did that.

Reply:
Leaving this then?

Reply:
Yes.